flepimop/gempyor_pkg/src/gempyor/compartments.py

```diff
@@ -849,13 +920,41 @@ def filter_func(lst, this_filter=[]):
         src.render(output_file)
 
 
-def get_list_dimension(thing):
+def get_list_dimension(thing: any) -> int:
```

Comment:
```suggestion
def get_list_dimension(thing: Any) -> int:
```

Lowercase `any` is a builtin function, uppercase `Any` is an object from `typing` to represent anything.